stable/defaultstorageclass/values.yaml

```diff
@@ -0,0 +1,2 @@
+issuer:
```

Comment:
`issuer` ?

Reply:
the `cert-manager` certificate for the webhook needs an `cert-manager` issuer. what do you suggest?

Reply:
are you gonna deploy a cert-manager pod ?

Reply:
no, the addon will be dependent on `cert-manager`

Reply:
`test.sh` deploys `cert-manager` just for testing purposes.